### PR TITLE
stability: fix fc name in hypervisor kill tests

### PR DESCRIPTION
**Pull request by bergwolf**
Instead of taking the full path name of the hypervisors, we need to
change to use only the name of the process because with jailer we have
different path shown there.

Fixes: #1846

### CODE REVIEW

File: integration/stability/hypervisor_stability_kill_test.sh [hunk 1]
```diff
@@ -21,18 +21,20 @@ PAYLOAD_ARGS="${PAYLOAD_ARGS:-tail -f /dev/null}"
 # Set the runtime if not set already
 RUNTIME="${RUNTIME:-kata-runtime}"
 
+HYPERVISOR_NAME=$(basename ${HYPERVISOR_PATH})
```
Comment (grahamwhaley) [inline integration/stability/hypervisor_stability_kill_test.sh]: nit: you can use shell magic here like `${HYPERVISOR_PATH%/*}` or similar ;-)

Comment (bergwolf) [reply integration/stability/hypervisor_stability_kill_test.sh]: Wow, that is real magic ;)

**Comment by bergwolf** [comment]
Because tests repo doesn't validate fc in CI, I'm trying to test it in a runtime PR, see https://github.com/kata-containers/runtime/pull/1905

**Comment by bergwolf** [comment]
@grahamwhaley It's the same issue but in a different test case -- the one I just added to killing hypervisor process.

**Comment by chavafg** [comment]
@GabyCT any idea of why fc job is not being triggered here?

**Comment by GabyCT** [comment]
@chavafg I can see that now is being triggered

**Comment by chavafg** [comment]
thanks @GabyCT, I see it now 😄

**Comment by GabyCT** [comment]
@bergwolf , if you want to launch the `fc` CI only you can use `/test-fc` to run only that job or '/test` and will run all the CI

**Pull request merged**